ansible/roles/ocp-workload-odo-workshop/tasks/pre_workload.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Copy .kube/config and set env var
+  copy:
+    src: ~/.kube/config
+    dest: /tmp/${GUID}
+  environment: 
+    KUBECONFIG: /tmp/${GUID}
```

Comment:
`${GUID}` is syntax error here. It's jinja template, not Shell.

Plus, environment is useless here.

I would simply add a variable in `defaults/main.yml`:

```yaml
tmp_kubeconfig: /tmp/{{ guid }}
```
Your `copy` task would become:

```yaml
- name: Copy .kube/config and set env var
  copy:
    src: ~/.kube/config
    dest: "{{ tmp_kubeconfig }}"
```
Then you have to add this part:

```
  environment:
    KUBECONFIG: "{{ tmp_kubeconfig }}"
```
to every tasks that run oc commands.

You can factorize using `block` like this:

```yaml
- environment:
    KUBECONFIG: "{{ tmp_kubeconfig }}"
  block:
     - task1
     - task2
     ...
```